treeherder/etl/management/commands/pulse_listener.py

```diff
@@ -0,0 +1,38 @@
+import environ
+from django.core.management.base import BaseCommand
+
+from treeherder.services.pulse import (JointConsumer,
+                                       prepare_joint_consumers)
+
+env = environ.Env()
+
+
+class Command(BaseCommand):
+    """
+    Management command to read tasks and pushes from a set of pulse exchanges.
+    This adds the pushes to a celery queue called ```store_tasks_pushes``` and
+    ```store_pulse_pushes```which does the actual storing of the pushes
+    in the database.
+    """
+    help = "Read tasks and pushes from a set of pulse exchanges and queue for ingestion"
+
+    def handle(self, *args, **options):
+        # Specifies the Pulse services from which Treeherder will ingest push
+        # information.  Sources can include properties `hgmo`, `github`, or both, to
+        # listen to events from those sources.  The value is a JSON array of the form
+        # [{pulse_url: .., hgmo: true, root_url: ..}, ..]
+        pulse_sources = env.json(
+            "PULSE_SOURCES",
+            default=[{"root_url": "https://firefox-ci-tc.services.mozilla.com",
+                      "github": True,
+                      "hgmo": True,
```

Comment:
Add `tasks` as a 3rd option instead of using `pulse_url` as a 3rd option (see comment on `treeherder/services/pulse/consumers.py`). That way a user could locally configure if they want to consume pushes or tasks.

Reply:
Naming it to tasks, we will have to make changes to ```self.connection = Connection(source['pulse_url'])``` in ```consumers.py``` which will need changes in both pulse_listener_tasks and pulse_listener_pushes, unless we intend to completely remove these two scripts

Reply:
I'm not asking to rename `pulse_url`. I'm asking for a new property to be added.